README.md

```diff
@@ -1197,6 +1197,14 @@ https://twitter.com/simalexan
 Topics: Serverless, Offline Web, Chat Bots\
 [https://twitter.com/slobodan\_](https://twitter.com/slobodan_)
 
+### Novi Sad
+
+<img src="https://avatars.githubusercontent.com/u/20664330?v=4" height="70px" width="70px" align="left" alt="" />
```

Comment:
we use the image that is associated with the link below, so switched this to a Twitter avatar too

```suggestion
<img src="https://res.cloudinary.com/dsscw65fc/image/twitter_name/stn1slv" height="70px" width="70px" align="left" alt="" />
```